src/sbt-test/test-projects/multi-module-project/changes/6.log

```diff
@@ -1 +1,2 @@
-src/libs/testKit/build.sbt
\ No newline at end of file
+src/libs/testKit/build.sbt
+src/libs/lib-1/src/test/scala/Test.scala
```

Comment:
Just add editorconfig from atlas, please.

Reply:
It was there but for some reason was disabled for this project in IDEA.